src/styles/style.css

```diff
@@ -0,0 +1,111 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&family=Montserrat&display=swap');
+
+html, body {
+    margin: 0;
+    padding: 0;
+    background-color: #01223A;

```

Comment:
we should have one global styles file with colors defined (so we don't repeat them)